src/test/java/org/jabref/preferences/JabRefPreferencesTest.java

```diff
@@ -0,0 +1,21 @@
+package org.jabref.preferences;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class JabRefPreferencesTest {
+
+    JabRefPreferences jabRefPreferences = new JabRefPreferences();
+
+    @Test
+    void getUserAndHostNotNull() {
+        assertNotNull(jabRefPreferences.getUserAndHost());
+    }
+
+    @Test
+    void getUserAndHostContainsHostname() {
+        assertTrue(jabRefPreferences.getUserAndHost().contains("-"));
+    }
```

Comment:
Are these really meaningful tests, or are they just there, so we have tested something, even if it does not really test something substantially meaningful. So getUserAndHost is not null? The second test does not test if it contains a hostname, just if it contains a `-` char. Just Maculature imho.